angular/src/lib/data-table/data-table.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
```

Comment:
@daanishnasir - This is the only file that has changed for this PR.  Apologies, this work was done off a branch of my preceding changes, and since that has not merged yet, you are seeing both sets of PRs changes.  Once the other pushes, hopefully, you will just see this file.